portal.go

```diff
@@ -985,6 +987,19 @@ func (portal *Portal) handleMessage(source *User, evt *events.Message, historica
 		if len(eventID) != 0 {
 			portal.finishHandling(existingMsg, &evt.Info, eventID, intent.UserID, dbMsgType, galleryPart, converted.Error)
 		}
+
+		// post_reply message is a WhatsApp status reply
+		if msgType == "post_reply" {
+			converted_quoted_message := portal.convertMessage(intent, source, &evt.Info, evt.Message.GetExtendedTextMessage().GetContextInfo().GetQuotedMessage(), false)
+			if converted_quoted_message == nil {
+				portal.log.Warnfln("Failed to convert post_reply message")
+				return
+			}
+			_, err := portal.sendMessage(converted_quoted_message.Intent, converted_quoted_message.Type, converted_quoted_message.Content, converted_quoted_message.Extra, evt.Info.Timestamp.UnixMilli())
```

Comment:
This looks like it'll send the message as the user who is replying rather than the user being replied to, and it won't include any indication that it's a reply specifically